extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -435,34 +458,46 @@ export class CreateSqlMigrationServiceDialog {
 		}).component();
 
 		this._setupContainer.display = 'none';
+		this._testConnectionButton.hidden = true;
 		return this._setupContainer;
 	}
 
 	private async refreshStatus(): Promise<void> {
-		const subscription = this.migrationStateModel._targetSubscription;
-		const resourceGroup = (this.migrationServiceResourceGroupDropdown.value as azdata.CategoryValue).name;
-		const location = this.migrationStateModel._targetServerInstance.location;
-		const migrationServiceStatus = await getSqlMigrationService(this.migrationStateModel._azureAccount, subscription, resourceGroup, location, this.createdMigrationService!.name);
-		const migrationServiceMonitoringStatus = await getSqlMigrationServiceMonitoringData(this.migrationStateModel._azureAccount, subscription, resourceGroup, location, this.createdMigrationService!.name);
-		this.createdMigrationServiceNodeNames = migrationServiceMonitoringStatus.nodes.map((node) => {
+		const subscription = this._model._targetSubscription;
+		const resourceGroup = (this.migrationServiceResourceGroupDropdown.value as azdata.CategoryValue).displayName;
+		const location = this._model._targetServerInstance.location;
+
+		const maxRetries = 5;
```

Comment:
Consider putting this refresh logic into a separate function that you can re-use since you're already using it in at least 2 places